src/libAtomVM/module.h

```diff
@@ -116,6 +116,7 @@ struct Module
     struct ListHead line_ref_offsets;
 
     const struct ExportedFunction **imported_funcs;
+    int functions_count;
```

Comment:
For the sake of saving sizeof(int) for every module, I would argue for recomputing this when the module is destroyed, especially considering we don't destroy modules at all for now.